hello-react/package.json

```diff
@@ -13,7 +13,9 @@
     "react-redux": "^7.2.8",
     "react-router-dom": "^6.3.0",
     "react-scripts": "5.0.0",
-    "redux-logger": "^3.0.6"
+    "redux": "^4.1.2",
+    "redux-logger": "^3.0.6",
+    "redux-thunk": "^2.4.1"
   },
   "scripts": {
     "start": "PORT=4000 react-scripts start",
```

Comment:
- [ ] [OPTIONAL] this command would only work in  a UNIX environment where you can set variables like this directly. I would recommend that you use an npm package like `cross-env` to ensure setting this variable works on a windows machine.